online_meetings/bluejeans_report.py

```diff
@@ -45,9 +45,9 @@
 token_json = json.loads(response)
 token = token_json["access_token"]
 logger.info(token)
-## the expiration time in seconds
+# the expiration time in seconds
 expires_in_sec = token_json["expires_in"]
-## minus for one minute, for safty check
+# minus for one minute, for safty check
```

Comment:
`safety`?

Reply:
Yeah, I think I noticed this before but didn't fix it. Just going through these files because of flake8 checking them now. Will update.